.github/workflows/checks.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           # the pushing fail
           pull_strategy: "NO-PULL"
 
```

Comment:
The "build" check actually just checked that the candid builds work and are up to date.  I have renamed it accordingly and added a new test that does the equivalent for protobuf.